readme.md

```diff
@@ -81,7 +88,8 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+Student.create(id: 1, name: "Geraldo")
+Sandwich.create(type: "pbj", student_id: 1)
 
```

Comment:
This is mostly right, however in SQL id's are serialized upon entry into database. It's kind of an anti pattern to hardcode in an `id` Instead, you could do something like this:

```ruby
geraldo = Student.create(name: "Geraldo")
Sandwich.create(type: "pbj", student: geraldo)
```